aspnet/fundamentals/routing.rst

```diff
@@ -1,10 +1,272 @@
-.. include:: /../common/stub-topic.txt
+Routing
+=======
+By `Steve Smith`_
 
-.. _routing-index:
+Routing middleware is used to map requests to route handlers. Routes are configured when the application starts up, and can define values that will be passed as parameters to route handlers. Routing functionality is also responsible generating links that correspond to routes in ASP.NET apps.
```

Comment:
Added a Recommendations section to the end. We can revise it as part of this PR or via post-publication issues/PRs.